R/zzz.R

```diff
@@ -138,16 +137,16 @@
       synBuildTable(name, parent, file)
     }
   )
-
+  #setGeneric("as.data.frame.cust", function(x) standardGeneric("as.data.frame.cust"))
```

Comment:
Is this comment needed?

Reply:
Nope, this is a testing line. I will remove it.